packages/scipy/test_scipy.py

```diff
@@ -44,6 +46,7 @@ def test_binom_ppf():
     assert binom.ppf(0.9, 1000, 0.1) == 112
 
 
+@pytest.mark.driver_timeout(40)
```

Comment:
I'm a bit confused about why this change caused `test_scipy_pytest` to start timing out.

Reply:
It seems like the driver_timeout update in other tests was affecting the timeout of `test_scipy_pytest` before this PR (I don't understand why :/). `test_scipy_pytest` always took more than 20 seconds.

Reply:
Yeah this must be correct, but I can't figure out why it was happening either.